src/cls/cas/cls_cas_client.h

```diff
@@ -17,17 +17,20 @@ void cls_cas_chunk_create_or_get_ref(
   librados::ObjectWriteOperation& op,
   const hobject_t& soid,
   const bufferlist& data,
-  bool verify=false);
+  bool verify=false,
+  int count = -1);
```

Comment:
The server side asserts that count isn't -1.  When is it valid to leave this as the default?